api/v1alpha1/pluggableprovider_types.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2025
```

Comment:
I will do the proper CR later. At the moment, I just wanted to point, that the CR has been added manually and not via the `kubebuilder`, hence the `PROJECT` now lacks the new CR. I suggest at least adding it manually with according settings if there are any validations/mutations or whether it is cluster-wide.

Reply:
Now there is divergance: the CR is cluster-scoped though the PROJECT file states opposite